TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Models/Person.cs

```diff
@@ -394,6 +396,56 @@ public bool InductionStatusManagedByCpd(DateOnly now)
         return (CpdInductionModifiedOn is not null && (InductionCompletedDate is null || InductionCompletedDate > sevenYearsAgo));
     }
 
+    public bool RefreshProfessionalStatusAttributes(
+        ProfessionalStatusType professionalStatusType,
+        IReadOnlyCollection<RouteToProfessionalStatus> allRoutes,
+        IEnumerable<ProfessionalStatus>? professionalStatusesHint = null)
```

Comment:
This parameter is here for when we're adding a professional status; when this method is called `this.Qualifications` won't have the new professional status in it yet.